### ISSUE TITLE
base64 sometimes squashing separate lines

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#299]
At present, if you make the WeBWorK sample chapter into PG files, the last problem (a problem all about tables) works fine if you examine its code or upload onto WeBWorK.

However if you make the HTML, there is a cell in the table that should be demonstrating cell/line behavior, yet puts the two lines onto the same line. Reverse translating the base64 string indicates that what should be:

```
    [[PF('[`\text{I}+\text{I}`]  
[`{}+\text{I}+\text{I}`]  
'),
```

with two spaces at the end of the first two lines (PGML's line break) is being translated as

```
    [[PF('[`\text{I}+\text{I}`][`{}+\text{I}+\text{I}`]'),
```

Without the space-space-linebreak, PGML does not know to put in a line break. This appears to be a base64 translation issue, since the direct PG output appears to be just fine.

**Comment by Alex-Jordan** [comment]
Not a base64 issue. Essentially same issue as #401, due to `webwork//table/cell/line` tying for priority with `table/cell/line` in `mathbook-html.xsl`. Being addressed with #438.

**Comment by rbeezer** [comment]
Should be fixed now as a result of #438
